_cloudstor/05-cloudstor-security.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Security Considerations
+order: 05
+duration: 5
+---
+
+### Your Password
+
+AARNET Cloudstor is secured using a password. Our recommendation is that you use the `App Password` that Cloudstor can generate for you. Please refer to the previous step for how to generate password. If you prefer to choose your own password, make sure your password meets minimum security standanrd.
+
+###  Snapshotting
+
+The setup of Cloudstor will store on your behalf in a privileged user account on your virtual machine. If you want to snapshot the virtual machine, the password will become part of the snapshot. If you subsequently share the snapshot with another user, you are also sharing the password, and the other user will have access to your data.
```

Comment:
The `cloudstor-setup` script will store your CloudStor password in plain text at /etc/davfs2/secrets `< need to check this is accurate!!`. If you decide to share a snapshot of your virtual machine with another user, this password will also be shared with them. If you intend on sharing snapshots, you should make arrangements to remove your CloudStor credentials before you start the snapshot process.

Reply:
Checked that the password is stored in plain text at /etc/davfs2/secrets